src/types/redis_hash.cc

```diff
@@ -217,7 +217,11 @@ rocksdb::Status Hash::Delete(const Slice &user_key, const std::vector<Slice> &fi
   if (!s.ok()) return s.IsNotFound() ? rocksdb::Status::OK() : s;
 
   std::string sub_key, value;
+  std::unordered_set<std::string> field_set;
```

Comment:
Why do we store std::string here rather than Slice? I think it is unnecessary to store a copy of string.

Reply:
Slice is not supported to use as type in std::unordered_set or std::set. Any suggestions to solve it?

Reply:
Personally I think `std::set` might works. `std::unordered_set` requires a `std::hash` or self-defined hash function.

Reply:
You can use std::string_view instead of Slice.